recipes-overlayed/kselftests/kselftests-next_git.bb

```diff
@@ -21,3 +21,4 @@ S = "${WORKDIR}/git"
 export INSTALL_PATH = "/opt/kselftests/next"
 
 require kselftests-common.inc
+RDEPENDS_${PN}_append = " python3-core "
```

Comment:
RDEPENDS_${PN} += "python3-core" is the more common form for this. Can you have a look at test_offload.py to see if it needs any modules python3-core doesn't drag in automatically?